MetaMask/metamask-extension
# Feat/notifications read all button

## **Description**

This PR introduces a "Mark all as read" button on the notifications page. By clicking it, the user can mark all notifications as read.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24687?quickstart=1)

## **Related issues**

## **Manual testing steps**

1. Go to the notifications page
2. Check if the button is present
3. Verify if, by clicking it, the notifications are marked as read

## **Screenshots/Recordings**

Check the Figma here https://www.figma.com/design/c7GgNw2kScGrVyRGAPhwEd/Notifications?node-id=6246%3A27186&t=kh6HjFvIDMmUe61Q-1

### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
